apps/Starship Prompt/install

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
+#Be sure to use the "error" function - it will display a message if a command fails to run. Example below:
+
+mkdir ~/.local/share/fonts/fira-code
+git_clone https://github.com/open-sorcerer64/fira-code-fonts.git || error 'Failed to download fonts'
+cd ~/fira-code-fonts || error 'Failed to move fonts'
+mv *.ttf ~/.local/share/fonts/fira-code || error 'Failed to move fonts'
+mv *.otf ~/.local/share/fonts/fira-code || error 'Failed to move fonts'
+
+echo fontname >> ~/.config/lxterminal/lxterminal.conf
+sed -i '/fontname/c\fontname=Fira Code Medium Nerd Font 11' ~/.config/lxterminal/lxterminal.conf
+echo FontName >> ~/.config/xfce4/terminal/terminalrc
+sed -i '/FontName/c\FontName=Fira Code Medium Nerd Font 12' ~/.config/xfce4/terminal/terminalrc
+
+sh -c "$(curl -fsSL https://starship.rs/install.sh)" || error 'Failed to download starship prompt'
+echo  eval "$(starship init bash)" >> ~/.bashrc || error 'Failed to set starship prompt'
```

Comment:
this does not work. did you test it? the eval attempts to run here, you want to place this at the end of the .bashrc, so you need to put it in ' '

```
echo 'eval "$(starship init bash)"' >> >> ~/.bashrc || error 'Failed to set starship prompt'
```
also this is dangerous if run twice, you should be removing all instances of this from .bashrc before adding it once

Reply:
see below post, you should do this when adding the line:

```
# remove all occurances
sed -i '/\eval "$(starship init bash)"/d' "$HOME/.bashrc"
# add one occurance to the .bashrc file
echo 'eval "$(starship init bash)"' >> ~/.bashrc || error 'Failed to set starship prompt'
```